python/src/main/db.py

```diff
@@ -1,8 +1,3 @@
-from python.src.main.Item import Item
-from python.src.main.Person import Person
-from python.src.main.parser import Parser
-
-
```

Comment:
db should only know about its interactions. It should not be responsible for doing any data transformation. It only wants to do gets, deletes and inserts.